qterminal_drop.desktop

```diff
@@ -55,6 +56,7 @@ GenericName[zh_TW]=下拉式終端機
 
 Comment=A drop-down terminal emulator.
 Comment[ca]=Un emulador de terminal desplegable.
+Comment[ca]=Vysouvací emulátor terminálu.
```

Comment:
Should be `Comment[cs]`?

Reply:
Yes, definitely – forgot to edit, sorry for that…